src/program/program.service.ts

```diff
@@ -668,8 +673,15 @@ export class ProgramService {
 		return this.prisma.assignment.create({
 			data: {
 				name: input.name,
-				moduleId: input.module,
-				dueAt: input.dueAt
+				module: {
+					connect: {
+						id: input.module
+					}
+				},
+				dueAt: input.dueAt,
+				contentType: "",
```

Comment:
This needs to be change to be the value of the respective property from the input object